ui/imports/shared/controls/chat/EmojiReaction.qml

```diff
@@ -2,32 +2,33 @@ import QtQuick
 
 import StatusQ.Core
 import StatusQ.Core.Theme
+import StatusQ.Components
 
 import utils
 import shared
 import shared.panels
 
 Rectangle {
-    property alias source: reactionImage.source
-    property string emoji
+    required property string emojiId
     property bool reactedByUser: false
     property bool isHovered: false
-    signal closeModal()
+    signal toggleReaction()
 
     id: root
-    width: reactionImage.width + Theme.halfPadding
+    width: statusEmoji.width + Theme.halfPadding
     height: width
     color: reactedByUser ? Theme.palette.secondaryBackground :
                            (isHovered ? Theme.palette.backgroundHover : Theme.palette.transparent)
     border.width: reactedByUser ? 1 : 0
     border.color: Theme.palette.primaryColor1
     radius: Theme.radius
 
-    SVGImage {
-        id: reactionImage
-        width: 32
-        fillMode: Image.PreserveAspectFit
+    StatusEmoji {
+        id: statusEmoji
         anchors.centerIn: parent
+        width: 20
```

Comment:
Do we want a fixed size or something dynamic for the times coming? ;)

Reply:
I followed the design, but you're right, the design was for Desktop only. What do you suggest to make it more dynamic?

Reply:
Fixed value is far from ideal but the question is what should be the base for that size. Maybe it could be derived from some font size (from Theme of), done using `FontMentrics`?